.haxerc

```diff
@@ -1,4 +1,4 @@
 {
-  "version": "b1b235b",
```

Comment:
Hmm, could have been nice to have a443fd3 (though not a big deal here)
Will try to generate that version